src/main/java/FrameworksDrivers/ScheduleGraphics.java

```diff
@@ -166,11 +149,10 @@ public void paint(Graphics g) {
 
```

Comment:
could working with colours be something we "factor out" into a helper method or otherwise store somewhere else, so that this paint method can be shorter?